webpack.config.js

```diff
@@ -92,7 +92,7 @@ module.exports = ({ production, extractCss, analyze, tests, hmr, port, host, } =
     path: outDir,
     publicPath: baseUrl,
     filename: production ? '[name].[chunkhash].bundle.js' : '[name].[fullhash].bundle.js',
-    sourceMapFilename: production ? '[name].[chunkhash].bundle.map' : '[name].[fullhash].bundle.map',
+    sourceMapFilename: production ? '[name].[chunkhash].bundle.map' : undefined,
```

Comment:
I compared with the aurelia-cli `webpack.config.js` file, and this was the difference (the cli one did not specify `sourceMapFilename` at all.

@dkent600 Do you have more insights (than me) about usage of `sourceMapFilename` in dev mode?

I tried to `throw new Error()` to see if we still get the filename in the trace.
Seems like we still do:
![grafik](https://user-images.githubusercontent.com/30693990/154771202-85736038-c129-48b6-8e68-4641e8bba43d.png)

Reply:
no, sorry